traffic_portal/test/integration/Data/deliveryservices.ts

```diff
@@ -357,17 +357,17 @@ export const deliveryservices = {
 				{
 					description: "delete a delivery service",
 					Name: "tpdservice1",
-					validationMessage: "Delivery service [ tpdservice1 ] deleted"
+					validationMessage: "Delivery service [ tpdservice1 ] deleted."
```

Comment:
Do you think it's better with the period? I think a fair few of the Alerts we send from the API have punctuation, but also a lot don't. To be fair, the actual Alert TO sends has one.

Reply:
I think in this case a period makes sense since there are two messages, even though they are as two separate level.